src/AppBundle/Markdown/UrlParser.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Markdown;
+
+use League\CommonMark\Inline\Element\Link;
+use League\CommonMark\Inline\Parser\AbstractInlineParser;
+use League\CommonMark\InlineParserContext;
+
+/**
+ * @author David Badura <d.a.badura@gmail.com>
+ */
+class UrlParser extends AbstractInlineParser
+{
+    const PATTERN = '#(?i)\b((?:[a-z][\w-]+:(?:/{1,3}|[a-z0-9%])|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:\'".,<>?«»“”‘’]))#iS';
+
+    /**
+     * @return string[]
+     */
+    public function getCharacters()
```

Comment:
yes, if i add the character `w` into the list, then www.some-domain.com will be detected. but to be a absolute path i need to prefix it with `http://` or `//`